assets/js/main.js

```diff
@@ -178,3 +87,108 @@ $(document).ready(
                 });
             });
     });
+
+// Export comment functionality and include functionality for comment count button
+function commentFunction() {
```

Comment:
@flowersw This is nitpicky, but I wouldn't include the word "function" in a function name. You might call it something like `enableCommentForm()` or something else that describes what it does a little more